pkg/controller/petset/pet.go

```diff
@@ -249,7 +251,8 @@ func (p *apiServerPetClient) SyncPVCs(pet *pcb) error {
 	for i, pvc := range pet.pvcs {
 		_, exists, err := p.getPVC(pvc.Name, pet.parent.Namespace)
 		if !exists {
-			if err := p.createPVC(&pet.pvcs[i]); err != nil {
+			var err error
+			if err = p.createPVC(&pet.pvcs[i]); err != nil {
 				errMsg += fmt.Sprintf("Failed to create %v: %v", pvc.Name, err)
 			}
 			p.event(pet.parent, "Create", fmt.Sprintf("pvc: %v", pvc.Name), err)
```

Comment:
Actually i think event() will check if err is nil internally, so you can probably get away with just:
```
err := p.createPVC()
p.event(parent, create, "pvc: name", err)
```